administrator/components/com_scheduler/src/Table/TaskTable.php

```diff
@@ -167,6 +167,44 @@ protected function _getAssetName(): string
         return 'com_scheduler.task.' . (int) $this->$k;
     }
 
+    /**
+     * Method to return the title to use for the asset table.
+     *
+     * @return  string
+     *
+     * @since   __DEPLOY_SINCE__
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```
@brianteeman I've never heard of `__DEPLOY_SINCE__`. I guess you meant `__DEPLOY_VERSION__`.